packages/ts-migrate-plugins/src/plugins/react-props.ts

```diff
@@ -28,11 +30,30 @@ const reactPropsPlugin: Plugin<Options> = {
     const updates: SourceTextUpdate[] = [];
     const getPropsTypeName = createPropsTypeNameGetter(sourceFile);
 
+    // Scan for prop type imports
+    const propTypeIdentifiers: PropTypesIdentifierMap = {};
+    for (const node of sourceFile.statements) {
+      if (ts.isImportDeclaration(node) && node.moduleSpecifier.getText() === '"prop-types"') {
```

Comment:
@arthuralee Imports are usually is the first statements of the file, maybe we can combine 2 loops into the one with a comment? 
This seems a little bit hacky, but it might prevent from doing 2nd iteration.

Reply:
Yeah, i used 2 loops to be 100% correct but you're probably right. will update